urdf2mjcf/postprocess/add_sensors.py

```diff
@@ -171,22 +205,12 @@ def add_sensors(site_name: str) -> None:
 
     # Add force sensors.
     for fs in metadata.force_sensors:
-        # Find the body to attach the site to
-        link_body = mjcf_root.find(f".//body[@name='{fs.body_name}']")
-        if link_body is None:
-            options = [body.attrib["name"] for body in mjcf_root.findall(".//body")]
-            logger.warning(
-                f"Body '{fs.body_name}' not found for force sensor site '{fs.site_name}'. "
-                f"Skipping sensor. Available bodies: {options}"
-            )
+        try:
+            site_elem = find_site(body_name=fs.body_name, site_name=fs.site_name, site_metadata=metadata.sites)
+        except BodyNotFoundError as e:
+            logger.warning(f"Skipping force sensor {fs.name} on body {fs.body_name} and site {fs.site_name}: {e}")
             continue
```

Comment:
i think it would be better to throw this error, rather than suppress it. logging messages are really easy to ignore